### PR TITLE
CORE-565 Making the docker experience more user friendly.

### PR DESCRIPTION
**Pull request by dzajkowski**
## Overview
Introduces a notion of profile. This should be considered a first step towards configurations (CORE-512)

### Does this PR relate to an RChain JIRA issue? 
https://rchain.atlassian.net/projects/CORE/issues/CORE-565

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
none.

### CODE REVIEW

File: node/src/main/scala/coop/rchain/node/Main.scala [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
-import scala.util.{Failure, Success, Try}
 
+import scala.util.{Failure, Success, Try}
 import org.bouncycastle.jce.provider.BouncyCastleProvider
 
 object Main {
 
   def main(args: Array[String]): Unit = {
     val conf = Conf(args)
+    println(s"Starting with profile ${conf.profile.toOption.map(_.name)}")
```
Comment (lukasz-golebiewski) [inline node/src/main/scala/coop/rchain/node/Main.scala]: Shouldn't this be logged at info level?

File: build.sbt [hunk 2]
```diff
@@ -155,8 +155,8 @@ lazy val node = (project in file("node"))
         Cmd("WORKDIR", (defaultLinuxInstallLocation in Docker).value),
         Cmd("ADD", s"--chown=$daemon:$daemon opt /opt"),
         Cmd("USER", daemon),
-        ExecCmd("ENTRYPOINT", "bin/rnode"),
-        ExecCmd("CMD", "run", "--data_dir=/var/lib/rnode")
```
Comment (lukasz-golebiewski) [inline build.sbt]: Wouldn't simply moving `"--data_dir=/var/lib/rnode"` to the line above work?

Comment (dzajkowski) [reply build.sbt]: `--data-dir` is an option on `run` not on `rnode`

File: node/src/main/scala/coop/rchain/node/Main.scala (hunk 1 above)
Comment (dzajkowski) [reply node/src/main/scala/coop/rchain/node/Main.scala]: from what I've seen the approach is to `println` the messages targeted at the user
vide:
https://github.com/rchain/rchain/blob/faceaf7c8c5f2407bd6ccf838220ba5be0b9bbaa/node/src/main/scala/coop/rchain/node/node.scala#L72

**Pull request merged**